shared/studio/tabs/queryEditor/index.tsx

```diff
@@ -149,7 +156,46 @@ export const QueryEditorView = observer(function QueryEditorView() {
         ]}
         state={editorState.splitView}
         minViewSize={20}
+        showFirstChildOnMobile={editorState.showQueryWindow}
       />
+      <div className={styles.mobileOverlayControls}>
+        <button
+          className={styles.mobileBtn}
+          onClick={() => editorState.setShowHistory(true)}
+        >
+          <MobileHistoryIcon className={styles.mobileHistoryIcon} />
+        </button>
+
+        <LabelsSwitch
+          labels={["query", "result"]}
+          value={
+            editorState.showQueryWindow ? switchState.left : switchState.right
+          }
+          onChange={() =>
+            editorState.setShowQueryWindow(!editorState.showQueryWindow)
+          }
+        />
+        <button
+          className={styles.mobileBtn}
+          onClick={() => editorState.runQuery()}
+          disabled={!editorState.canRunQuery}
+        >
+          <MobileRunIcon className={styles.mobileRunIcon} />
+        </button>
+      </div>
+      {editorState.showHistory && (
+        <div className={styles.mobileHistory}>
+          <p className={styles.title}>History</p>
+          <button
+            onClick={() => editorState.setShowHistory(false)}
+            className={styles.closeHistoryBtn}
+          >
+            <CrossIcon />
+          </button>
+
+          <HistoryPanel className={styles.historyPanel} />
```

Comment:
I think this bit needs hiding when not on mobile:
![image](https://github.com/edgedb/edgedb-ui/assets/2085287/61503e3b-58d3-483a-805a-c570d4de892e)

Reply:
Yes, I work on this. Saw it too. Tnx